src/index.html

```diff
@@ -9,6 +9,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-items">
+          <li><a class="header__nav-item is-active" href="#">Apple</a></li>
```

Comment:
1. `li` - is an item
`a` - is a link

2. apply code styling recommendations from a task readme
3. make these changes all over the page

```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="#">
              Apple
            </a>
          </li>
```